music21/metadata/primitives.py

```diff
@@ -1488,7 +1488,7 @@ def testDateSelection(self):
 
 
 ValueType = t.Union[DateSingle, DateRelative, DateBetween, DateSelection,
-                    Text, Contributor, Copyright]
+                    Text, Contributor, Copyright, int]
```

Comment:
where is "int" included in this?  It doesn't seem to make sense as part of this definition.  In the few places where an int can be returned would it be better to type that as `t.Union[ValueType, int]` -- otherwise you won't be able to do `x: ValueType = function();  print(x._data)` because ints don't have a ._data attribute.

Reply:
ValueType is "the types of the values that might be found in self._contents".  I didn't realize it also needed to mean "types with _data".  I found ValueType really useful (instead of using t.Any), and I'm not sure using t.Union[ValueType, int] everywhere will actually solve your problem (if function returns t.Union[ValueType, int], you still can't print(x._data).  Perhaps we shouldn't use int in _contents at all, and have a new type with an int x._data field?

Reply:
Oh, and int is in there because of fileNumber.